frontend/src/components/GameDetails/GamePropertiesCard/GamePropertiesCard.tsx

```diff
@@ -0,0 +1,74 @@
+import { useUser } from '@/src/hooks/api/auth';
+import Link from 'next/link';
+import { FC } from 'react';
+import Button from '../../Forms/Button/Button';
+import styles from './GamePropertiesCard.module.scss';
+
+interface GamePropertiesCardProps {
+	gameId: string;
+	owner: string;
+	platform: string;
+	location: string;
+	releaseDate: string;
+	playtime: number;
+	playerMax: number;
+	playerMin: number;
+	borrowed: boolean;
+}
+
+const GamePropertiesCard: FC<GamePropertiesCardProps> = ({
+	gameId,
+	owner,
+	platform,
+	location,
+	releaseDate,
+	playtime,
+	playerMax,
+	playerMin,
+	borrowed
+}) => {
+	return (
+		<div className={styles.card}>
+			<div>
+				<Property title="Owned by" value={owner} />
+				<Property title="Platform" value={platform} />
+				<Property title="Location" value={location} />
+				<Property title="Release Date" value={releaseDate} />
+				<Property title="Playtime" value={`${playtime} min`} />
+				<Property
+					title="Players"
+					value={
+						playerMax == playerMin
+							? `${playerMin}-${playerMax}`
+							: playerMin.toString()
+					}
+				/>
+				<Property
+					title="Availability"
+					value={borrowed ? 'Borrowed' : 'Not borrowed'}
+				/>
+			</div>
+			<BorrowGame gameId={gameId} />
+		</div>
```

Comment:
We should add the games average rating somewhere. Maybe here?

Reply:
Not actually part of the design that I worked from, so it's kind of @Krysset that needs to answer for this.